qns/Dockerfile

```diff
@@ -33,35 +34,22 @@ RUN set -eux; \
 
 RUN "$NSS_DIR"/build.sh --static -Ddisable_tests=1 -o
 
-# Copy the .git directory from the local clone so that it is possible to create
-# an image that includes local updates.
-RUN mkdir -p /neqo-reference
-ADD . /neqo-reference
-RUN if [ -d /neqo-reference/.git ]; then \
-      source=/neqo-reference; \
-    else \
-      source=https://github.com/mozilla/neqo; \
-    fi; \
-    git clone --depth 1 --branch "$NEQO_BRANCH" "$source" /neqo; \
-    rm -rf /neqo-reference
+ADD . /neqo
 
 RUN set -eux; \
     cd /neqo; \
     RUSTFLAGS="-g -C link-arg=-fuse-ld=lld" cargo build --release \
-      --bin neqo-client --bin neqo-server; \
-    cp target/release/neqo-client target; \
-    cp target/release/neqo-server target; \
-    rm -rf target/release
+      --bin neqo-client --bin neqo-server;
```

Comment:
```suggestion
      --bin neqo-client --bin neqo-server
```

Reply:
:+1: thanks for the catch! ae21e354d529448d2ccaf690005f21a5de73c531